src/TodoList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import TodoItem from './TodoItem';
+
+export default function TodoList({ todos, onClickDeleteTodo, onClickCheckTodo }) {
+  if (todos.length === 0) return <p>할 일이 없어요!</p>;
```

Comment:
```suggestion
  if (todos.length === 0) {
    return (
      <p>할 일이 없어요!</p>
    );
   }
```
슬랙에서 공유해드렸듯이 중괄호를 생략하고 한줄로 작성하는 형태는 추천드리지않아요.
특별한 사항이 없으면 기본 형태를 지켜주시는게 에러나 실수를 피할 확률을 높여줘요.